system/BaseModel.php

```diff
@@ -1291,10 +1291,10 @@ protected function intToDate(int $value)
                 return $value;
 
             case 'datetime':
-                return date('Y-m-d H:i:s', $value);
+                return Time::createFromTimestamp($value, app_timezone())->toDateTimeString();
```

Comment:
`toDateTimeString()` returns localized value. It is not good as data to save in database.
```suggestion
                return (string) Time::createFromTimestamp($value, app_timezone());
```